GettingStarted.md

```diff
@@ -22,10 +22,10 @@ For some of the supported package managers and SCMs additional tools need to be
 
 * CVS
 * Mercurial
-* NPM (required for this tutorial because we scan `mime-types` which is an NPM project)
+* Node JS version 8.x and NPM version 5.5.1 (required for this tutorial because we scan `mime-types` which is an NPM project)
 * SBT
 * Subversion
-* Yarn
+* Yarn (version 1.3.2)
```

Comment:
To align with the above wording, I'd drop the parentheses, and also the "version" and just say "Yarn 1.3.2".